apps/frontend/src/app/PageCharacter/CharacterDisplay/Tabs/TabOverview/EquipmentSection.tsx

```diff
@@ -88,9 +89,10 @@ function ArtSwapCard({ slotKey }: { slotKey: SlotKey }) {
   const { character: { key: characterKey } } = useContext(CharacterContext)
   const { database } = useContext(DatabaseContext)
   const [show, onOpen, onClose] = useBoolState()
+  const { t } = useTranslation("artifact")
   return <CardLight sx={{ height: "100%", width: "100%", minHeight: 300, display: "flex", flexDirection: "column" }}>
     <CardContent>
-      <Typography><SlotNameWithIcon slotKey={slotKey} /></Typography>
+      <Typography sx={{ display: "flex", gap: 1 }}><SlotIcon iconProps={iconInlineProps} slotKey={slotKey} /> {t<string>(`slotName.${slotKey}`)}</Typography>
```

Comment:
Also why is there a random `<string>` here? removing it didn't seem to hurt anything

Reply:
um, it explicitly casts the return of `t` as a `string`, which doesnt do much now since we disabled it from returning `null`

Reply:
I've removed all occurences.